tests/phpunit/Unit/ScribuntoLuaLibrarySetTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace SMW\Scribunto\Tests;
+
+/**
+ * @covers \SMW\Scribunto\ScribuntoLuaLibrary
+ * @group semantic-scribunto
+ *
+ * @license GNU GPL v2+
+ * @since 1.0
+ *
+ * @author oetterer
+ */
+class ScribuntoLuaLibrarySetTest extends ScribuntoLuaEngineTestBase {
+
+	/**
+	 * Lua test module
+	 * @var string
+	 */
+	protected static $moduleName = self::class;
+
+	/**
+	 * ScribuntoLuaEngineTestBase::getTestModules
+	 */
+	public function getTestModules() {
+		return parent::getTestModules() + array(
+			self::$moduleName => __DIR__ . '/' . 'mw.smw.set.tests.lua',
+		);
+	}
```

Comment:
Maybe they can be put together (having an `array`). A possible consolidation should happen after this PR has been merged so you split re-factoring from re-designing and leave the tests intact when trying the change the registration system.